.evergreen/run-tests.sh

```diff
@@ -12,6 +12,7 @@ set -o errexit  # Exit the script with error if any of the commands fail
 AUTH=${AUTH:-noauth}
 UNIFIED=${UNIFIED:-}
 MONGODB_URI=${MONGODB_URI:-}
+TEST_NPM_SCRIPT="check:test"
```

Comment:
nit: do you actually use this variable anywhere? It doesn't look like its ever overridden.

Reply:
I missed porting over a few changes from `3.6`, done now. Sorry about the confusion!

This variable isn't strictly necessary; it is (now) used by the NO_EXIT flag logic, but that logic could easily be moved to the bottom where the npm script is executed.